### DISCUSSION TITLE
Is this plugin still maintained/under development?

### DISCUSSION THREAD

**Discussion by sparklekraken** [liamcain/obsidian-calendar-plugin#233] [General]
Looking at the change history on this repository it seems that the author has stopped updating during early 2021.
Is this still maintained?

**Comment by liamcain** [comment]
Hi, this plugin is not abandoned, but you're right, it has been a long time since it's seen any active development. Due to a combination of factors, I wasn't able to do much plugin development in the last year and I'm still working up the courage to tackle the backlog of issues for the Calendar plugin now.

I have a lot of larger ideas for the direction I want to take the plugin, but due to its popularity, it's a much trickier task. I've offered timelines in the past that I've failed to meet I won't promise when you'll see new features, but I'm hoping to ramp up on development again soon.

---

I do see you filed [an issue](https://github.com/liamcain/obsidian-calendar-plugin/issues/232) a couple days ago though. I'll take a look now 👀

**Reply by sparklekraken** [reply]
Hey mate, no problem at all! Fully understand that Life(TM) and other stuff happens. I just filed that issue then realised that there hadn't been much movement in the repo as a whole so wanted to check.

No rush on the issue I filed BTW, it's not a massive breaking bug as far as I can tell and it's not stopping me from using the plugin or Obsidian in general.

Thanks for taking the time to develop and maintain this and I absolutely understand the concern about setting deadlines etc that get missed because of totally valid reasons (I have the same issue!). So take all the time you need :)

**Reply by jarbochov** [reply]
Was wondering the same thing. Thanks for your communication.

**Comment by Tijger-T** [comment]
Eighteen-months further and as yet, still nothing new.
I freely admit the plug-in works without any real glitches and that bringing something up to date, particularly when there is a backlog _and_ a heap of new ideas – of which it is unclear which takes priority!!! – is not easy.
I don't wish to criticize Liam at all, just hoping he's still keeping at least one eye on the project 😊

**Reply by mattmaiorana** [reply]
Agreed. Would getting some Patreon money flowing help with making sure things are maintained? It's the fifth most installed plugin after all. I've noticed a few odd bugs - nothing catastrophic but still things I would love to see ironed out.

With 780k having this installed... seems like we could probably get a good chunk of money together if we all chipped in a few bucks? I use it enough that I would be happy to contribute. 

Or maybe this could get merged into a core feature...?
